release notes/v0.40.0.md

```diff
@@ -0,0 +1,253 @@
+k6 v0.40.0 is here! :tada: This release includes the promised experimental modules, some other small fixes and a lot of changes needed for the metric time series, unindexable tags and native ESM support. [Roadmap and future plans section](#roadmap-and-future-plans) makes another appearance with more info on how things are going and what should be expected
+
+# New Features and notable changes!
+
+## ⚠Breaking change⚠: Main module/script no longer taints the global scope [#2571](https://github.com/grafana/k6/pull/2571)
+
+As part of the ESM changes, it was found that anything defined in the main module's scope was also accessible globally. This was because it was directly evaluated in the global scope.
```

Comment:
```suggestion
## Main module/script no longer taints the global scope [#2571](https://github.com/grafana/k6/pull/2571)

[:warning: This is a breaking change]

[:warning: Breaking change] As part of the ESM changes, it was found that anything defined in the main module's scope was also accessible globally. This was because it was directly evaluated in the global scope.
```
Maybe moving it to the content text could be nicer, I added two suggestions but we can put it where we think it's better.

Reply:
[b358c6a](https://github.com/grafana/k6/pull/2669/commits/b358c6aa17f7f9e5eb613996f5516113422517ac)